code/Base/GamemodeSystem/GamemodeOrb.cs

```diff
@@ -64,8 +59,19 @@ public override void StartTouch( Entity other )
 
 		if ( other is PlazaPlayer player )
 		{
+			Log.Debug( $"Orb trying to add player: {player.Client.Name} to gamemode: {Gamemode}" );
 			LinkedGamemode?.AddClient( player.Client );
 			LastTouch = 0;
 		}
 	}
+
+	[Event.Tick]
```

Comment:
Take note that this will tick on the server & client at the same time.

Reply:
this was an oversight as I just wanted to cram debug draw in while the stupid particle wasn't showing haha, will adjust it